README.md

```diff
@@ -93,35 +337,36 @@ await consumer.run({
         }
       })
 
-      resolveOffset(message.offset)
+      await resolveOffset(message.offset)
+      await heartbeat()
     }
   },
 })
 
 // remember to close your consumer when you leave
+await consumer.disconnect()
 ```
 
-### Configure SSL and SASL
+#### <a name="consuming-messages-options"></a> Options
 
-```javascript
-const fs = require('fs')
-const kafka = new Kafka({
-  clientId: 'my-app',
-  brokers: ['kafka1:9092', 'kafka2:9092']
-  ssl: {
-    cert: fs.readFileSync('<path/to>/client_cert.pem', 'utf-8'),
-    key: fs.readFileSync('<path/to>/client_key.pem', 'utf-8'),
-    ca: [fs.readFileSync('<path/to>/ca_cert.pem', 'utf-8')],
-  },
-  sasl: {
-    mechanism: 'plain',
-    username: 'my-username',
-    password: 'my-password',
-  },
-})
-```
+- __createPartitionAssigner__ - default: `round robin`
+- __sessionTimeout__ - Timeout in milliseconds used to detect failures. The consumer sends periodic heartbeats to indicate its liveness to the broker. If no heartbeats are received by the broker before the expiration of this session timeout, then the broker will remove this consumer from the group and initiate a rebalance. default: `30000`
+- __heartbeatInterval__ - The expected time in milliseconds between heartbeats to the consumer coordinator. Heartbeats are used to ensure that the consumer's session stays active. The value must be set lower than session timeout. default: `3000`
```

Comment:
It would be good to include information on how to choose these values.

Reply:
Maybe on the next write up, I want to change the defaults as well. WDYT?